README.md

```diff
@@ -4,9 +4,9 @@ The platform assists member in managing expenses, tasks , voting, and more.
 
 ## ERD
 
-![ERD](https://user-images.githubusercontent.com/74178515/204143707-970216f5-e382-47d1-ab2e-209aa7aaa18b.png)
+![ERD](https://lucid.app/documents/embedded/af373954-3144-4325-9253-494a0af1512b?invitationId=inv_d92b11b0-d0dd-445e-98c2-8493166012ea#)
 
 ## DFD
 
-![DFD](https://user-images.githubusercontent.com/74178515/204143717-32b0ef58-5bdd-438f-9523-9184e3921f9e.png)
+![DFD](https://lucid.app/documents/embedded/10221f64-8c21-4dd6-a916-c735b66ae3d0?invitationId=inv_4c247743-b1be-484b-8689-75db7ffb786e#)
```

Comment:
About the DFD I think:
- The same as "join commune" you should have the option to leave/delete from the commune.
- The user can create/change the chore should be at the same level as the join commune.

Reply:
"The user can create/change the chore should be at the same level as the join commune."
What do mean by the same level?

Reply:
I think he mean like in the same column because it function that equal to the other functions  **but** I am  also think it should be after the vote is pass

Reply:
@NicolePauzner @ofirtzabari you can start voting only after you created the chore 
so as @ofirtzabari said in the "same" column - above the "complete chore" circle